configure

```diff
@@ -322,10 +323,19 @@ cat > "projects/illumos/illumos.sh" <<EOF
 NIGHTLY_OPTIONS="$nopts";			export NIGHTLY_OPTIONS
 GATE="${RELEASE_VER}";				export GATE
 CODEMGR_WS="$conf_root/projects/illumos";	export CODEMGR_WS
+# NOTE: build_illumos modifies this initial constant
 MAX_JOBS=128
 maxjobs() {
         ncpu=\`kstat -p cpu_info:::state | grep -c on-line\`
-        if [[ \$(( \$ncpu + 2 )) -lt \${MAX_JOBS} ]]; then
+	zcpucap=\`pfexec mdata-get sdc:cpu_cap\`
```

Comment:
While this seems to work as-is, if there is no cpu_cap value then zcpucap will be a zero length string, not the numeral `0`. There may be a subtle bug here waiting to lash out.

This also applies to tools/optimize_jobs below.

Reply:
Both will need fixing.